logbus/logstreamer/logstreamer.go

```diff
@@ -17,29 +20,149 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+const (
+	DefaultChSize = 10240
```

Comment:
Nit: Make private?

Reply:
I usually export defaults like this so that anyone who reads the `godoc` can see what the default should be. I could alternatively put it in a comment, but then we're in the `comments are lies waiting to happen` territory - it feels safer to just export the value. It's a constant, so there's no risk of anything external modifying it.

Reply:
Sure.. then make the other one public too? We do add comments for everything exported.

Reply:
Exported, commented. I also changed the name a bit to avoid referencing "channel" - since one of the main reasons for changing it to use an iterator interface is so that we can change the underlying channel to a different type if we want.